packages/ActionBar/src/hooks/useSort/useSort.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import nanoid from "nanoid";
+import sort from "./sort";
+
+function sortData({ sortedFields, columns, data }) {
+  let sortedData = data;
+  if (sortedFields.length > 0) {
+    sortedFields.forEach(field => {
+      sortedData = sort({
+        data,
+        columnId: field.columnId,
+        direction: field.direction,
+        columnType: columns.find(column => field.columnId === column.id).type,
+        momentParsingFormat: columns.find(column => field.columnId === column.id).momentParsingFormat,
+      });
+    });
```

Comment:
I don't quite understand how that works. If `sortedFields.length > 1` wouldn't `sortedData` be overridden ?

Reply:
Also, `columns.find(column => field.columnId === column.id)` can be extracted

Reply:
If there's 2 fields for different columns, I'll sort by the first field first and then sort the data again by the second field. So the results compliance with both sorted fields.
If there's 2 fields for the same columns, the second one will override the first one.